driver_with_mockserver_test.go

```diff
@@ -979,10 +979,14 @@ func nullDate(valid bool, v string) spanner.NullDate {
 }
 
 func setupTestDbConnection(t *testing.T) (db *sql.DB, server *testutil.MockedSpannerInMemTestServer, teardown func()) {
+	return setupTestDbConnectionWithParams(t, "")
+}
+
+func setupTestDbConnectionWithParams(t *testing.T, params string) (db *sql.DB, server *testutil.MockedSpannerInMemTestServer, teardown func()) {
```

Comment:
Should we use `DB` instead of `Db`?

Reply:
Good point, to follow the convention of go-sql, it should be `DB`.